main.cpp

```diff
@@ -71,6 +71,9 @@ int main()
     mbed_trace_print_function_set(trace_printer);
     mbed_trace_mutex_wait_function_set( serial_out_mutex_wait );
     mbed_trace_mutex_release_function_set( serial_out_mutex_release );
+    
+    printf("Start mesh-minimal application\n\r");
```

Comment:
Replace `\n\r` with just `\n`
We already have `platform.stdio-convert-newlines: true`  https://github.com/ARMmbed/mbed-os-example-mesh-minimal/blob/master/mbed_app.json#L27

(And also, it should have been `\r\n` not `\n\r`, remebers its **R**etur**N**, See https://en.wikipedia.org/wiki/Newline)